command/git.go

```diff
@@ -434,8 +437,15 @@ func (c *gitFetchProject) Execute(ctx context.Context,
 
 		err = jpm.CreateCommand(ctx).Add([]string{"bash", "-c", strings.Join(moduleCmds, "\n")}).
 			Directory(filepath.ToSlash(filepath.Join(conf.WorkDir, c.Directory))).
-			SetOutputSender(level.Info, logger.Task().GetSender()).SetErrorWriter(stdErr).Run(ctx)
+			SetOutputSender(level.Info, logger.Task().GetSender()).SetErrorWriter(stdErr).SetOutputWriter(stdOut).Run(ctx)
```

Comment:
From typing `git checkout HEAD~ 1> /dev/null` and `git checkout HEAD~ 2> /dev/null`, it looks like my initial assumption was wrong: It looks like the info we want is in fact in stderr:

```
$ git checkout HEAD~ 1> /dev/null
Previous HEAD position was 70db34e7 Fix another occurrence of tx api not clearing in progress flag (#1321)
HEAD is now at 2c5156f3 RUBY-1706 Support Retryable Writes on by Default (#1290)
```
I think therefore you don't need to add standard out to the output sender, but you should keep line 442 on, but with stderr instead of stdout.

Reply:
I didn't actually know about the stdout/stderr file descriptors. Thanks for laying it out here.